cmd/uhc/cluster/list/list.go

```diff
@@ -195,23 +262,34 @@ func run(cmd *cobra.Command, argv []string) {
 		managed = false
 	}
 
-	// DEFAULT PADDING, CHANGE THIS VALUE TO CLIP MORE OR LESS
+	// Assign padding and column name variables:
 	paddingByColumn := []int{
 		35,
 		40,
 		80,
 		35,
 		15,
 	}
-
-	// Pad and print column names + space between rows for asthetics
 	columnNames := []string{
 		"ID",
 		"Name",
 		"API URL",
 		"Version",
 		"Region",
 	}
```

Comment:
For `paddingByColumn` and `columnNames` I think it would be better to use maps, with the column name as the index:

```go
paddingByColumn := map[string]int{
        "id": 35,
        ...
}

columnNames := map[string]string{
        "id": "ID",
         ...
}
```

Reply:
No longer needed